File: Easy/1672.richest-customer-wealth.cpp
class Solution {
 public:
  int maximumWealth(vector<vector<int>> &accounts) {
    int max_wealth = 0;
    for (auto &customer : accounts) {
      int current_wealth = accumulate(customer.begin(), customer.end(), 0);
      if (current_wealth > max_wealth)
        max_wealth = current_wealth;
    }
    return max_wealth;
  }
};

File: rust/src/1672.richest-customer-wealth.rs
impl Solution {
  pub fn maximum_wealth(accounts : Vec<Vec<i32>>)->i32 {
    let mut max_wealth : i32 = 0;
    for
      i in 0..accounts.len() {
        let mut curr_wealth : i32 = 0;
      for
        j in 0..accounts [i].len() { curr_wealth += accounts[i][j]; }
      if max_wealth
        < curr_wealth { max_wealth = curr_wealth; }
      }
    max_wealth
  }
}
